AudioKitSynthOne/Sequencer/UI Components/RateKnob.swift

```diff
@@ -10,12 +10,12 @@ import UIKit
 
 public class RateKnob: MIDIKnob {
 
+    private var _value: Double = 0
```

Comment:
Can we avoid using `_` before variable name?

Reply:
done